File: Iniciante/1020/1020/1020.c
#include <stdio.h>

int main()
{
    int idade, anos, meses, dias;
    
    scanf("%d", &idade);
    
        anos = idade/365;
        meses = (idade - anos*365)/30;
        dias = (idade - anos*365 - meses*30); 
        
    printf ("%d ano(s)\n", anos);
    printf ("%d mes(es)\n", meses);
    printf ("%d dia(s)\n", dias);

    return 0;
}

File: Lua/Beginner/1020.lua
dias = io.read('*n')

anos = math.floor(dias/365)
dias = dias%365
meses = math.floor(dias/30)
dias = dias%30

print(string.format('%d ano(s)', anos))
print(string.format('%d mes(es)', meses))
print(string.format('%d dia(s)', dias))